### ISSUE TITLE
paper: should it be interpretation of linear relatedness?

### ISSUE THREAD

**Issue by rmflight** [mingzehuang/latentcor#8]
Starting on [Line 28 of the PDF of the paper draft](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03634/joss.03634/10.21105.joss.03634.pdf), I see this statement that seems to need some clarification:

> the resulting values do not have correlation interpretation and can not be used as direct substitutes in statistical methods that require correlation as input

Are you stating that they don't have a **linear correlation** interpretation, and therefore aren't appropriate for other methods? I would really like to see a reference for this claim, and rewording to make it clearer.

**Comment by muellsen** [comment]
Thanks a lot for the comment. To avoid any confusion about semantics or interpretation, we tried to make our statement more precise. The sentence reads now

While the rank-based Kendall’s τ and Spearman’s ρ are more robust measures of _association_,
they cannot directly be used as subsitutes for statistical methods that require Pearson correlation as input (a prominent example is, e.g., graphical model estimation (Xue & Zou, 2012;Yoon et al., 2019)).

Here, we also added the Xue & Zou reference. The paper succinctly describes the difficulty of using rank-based estimators in the context of graphical model estimation and provides a solution to it for continuous data.

More generally, rank-based estimators are not mere "generalizations" of Pearson correlation but, indeed, measure a different "type" of relationship (see, e.g., the nice example in Wikipedia's correlation article https://en.wikipedia.org/wiki/Correlation in the Subsection on Rank correlations (and references therein)). This is what we originally meant by "correlation interpretation" (but this was obviously too vague).
